tests/test_templates.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+import unittest
+
+import numpy as np
+
+from pint.templates import lcprimitives
+from pint.templates import lctemplate
+
+
+def gauss(x, x0, s):
+    return 1.0 / s / (2 * np.pi) ** 0.5 * np.exp(-0.5 * (x - x0) ** 2 / s ** 2)
+
+
+class TestPrimitives(unittest.TestCase):
+    """ Test instantiation and various properties of LCPrimitive objects.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    def test_basic_functionality(self):
```

Comment:
There's no need to create a class; we're using pytest so you can just create this as a function.